rkyv/src/time.rs

```diff
@@ -109,3 +110,13 @@ impl ArchivedDuration {
         addr_of_mut!((*out).nanos).write(to_archived!(nanos));
     }
 }
+
+impl PartialEq<Duration> for ArchivedDuration {
```

Comment:
When possible, we also want to implement the reverse relationship. In this case, it's `impl PartialEq<ArchivedDuration> for Duration`. It can just delegate to the other impl.